ic-os/components/setupos-scripts/setup-disk.sh

```diff
@@ -37,9 +37,22 @@ function purge_partitions() {
     for drive in "${large_drives[@]}"; do
         echo "Wiping partitions on drive: /dev/${drive}."
 
+        # Remove any LVM metadata
+        pvremove --force "/dev/${drive}"* 2>/dev/null || true
+        vgremove --force hostlvm 2>/dev/null || true
+
+        # Remove any device mappings
+        dmsetup remove_all 2>/dev/null || true
```

Comment:
Maybe we can remove this `vgremove`, and move the `dmsetup` below, since these are not changed by this loop.